cluster/validate-cluster.sh

```diff
@@ -51,7 +51,11 @@ ALLOWED_NOTREADY_NODES="${ALLOWED_NOTREADY_NODES:-0}"
 CLUSTER_READY_ADDITIONAL_TIME_SECONDS="${CLUSTER_READY_ADDITIONAL_TIME_SECONDS:-30}"
 
 if [[ "${KUBERNETES_PROVIDER:-}" == "gce" ]]; then
-  EXPECTED_NUM_NODES="$(get-num-nodes)"
+  if [[ "${KUBE_CREATE_NODES}" == "false" ]]; then
```

Comment:
Instead please check if == true. Otherwise, you treat e.g. "skdjfks" as true, whereas in util.sh you treat it as false.

Reply:
Done